src/data/markdown/docs/01 guides/04 Results visualization/06 JSON.md

```diff
@@ -100,147 +100,7 @@ For more advanced cases, check out the [jq Manual][jq_manual_url]
-
-```json
-{
-  "metrics": {
-    "checks": {
-      "fails": 0,
-      "passes": 434,
-      "value": 0
-    },
-    "data_received": {
-      "count": 4267088,
-      "rate": 213353.42114517145
-    },
-    "data_sent": {
-      "count": 33726,
-      "rate": 1686.2922633754104
-    },
-    "http_req_blocked": {
-      "avg": 2.2303510806451645,
-      "max": 466.009823,
-      "med": 0.006873000000000001,
-      "min": 0.001265,
-      "p(90)": 0.0093969,
-      "p(95)": 0.01104339999999999
-    },
-    "http_req_connecting": {
-      "avg": 1.4719108986175116,
-      "max": 176.017402,
-      "med": 0,
-      "min": 0,
-      "p(90)": 0,
-      "p(95)": 0
-    },
-    "http_req_duration": {
-      "avg": 146.13434929493093,
-      "max": 663.812323,
-      "med": 115.237707,
-      "min": 104.336176,
-      "p(90)": 204.6849077,
-      "p(95)": 205.27533795
-    },
-    "http_req_receiving": {
-      "avg": 0.32699811981566856,
-      "max": 5.86652,
-      "med": 0.135682,
-      "min": 0.026588,
-      "p(90)": 1.4118543,
-      "p(95)": 1.6629313999999984
-    },
-    "http_req_sending": {
-      "avg": 0.031275672811059876,
-      "max": 0.149365,
-      "med": 0.028853,
-      "min": 0.006069,
-      "p(90)": 0.041769099999999997,
-      "p(95)": 0.05106769999999998
-    },
-    "http_req_tls_handshaking": {
-      "avg": 0,
-      "max": 0,
-      "med": 0,
-      "min": 0,
-      "p(90)": 0,
-      "p(95)": 0
-    },
-    "http_req_waiting": {
-      "avg": 145.77607550230408,
-      "max": 663.551133,
-      "med": 114.90324000000001,
-      "min": 103.389659,
-      "p(90)": 204.4926965,
-      "p(95)": 205.02902265
-    },
-    "http_reqs": {
-      "count": 434,
-      "rate": 21.699900441941768
-    },
-    "iteration_duration": {
-      "avg": 148.59040099078345,
-      "max": 671.970894,
-      "med": 115.69334950000001,
-      "min": 104.558313,
-      "p(90)": 205.0391277,
-      "p(95)": 209.87447475000002
-    },
-    "iterations": {
-      "count": 434,
-      "rate": 21.699900441941768
-    },
-    "vus": {
-      "max": 5,
-      "min": 1,
-      "value": 1
-    },
-    "vus_max": {
-      "max": 5,
-      "min": 5,
-      "value": 5
-    }
-  },
-  "root_group": {
-    "name": "",
-    "path": "",
-    "id": "d41d8cd98f00b204e9800998ecf8427e",
-    "groups": {},
-    "checks": {
-      "status is 200": {
-        "name": "status is 200",
-        "path": "::status is 200",
-        "id": "6210a8cd14cd70477eba5c5e4cb3fb5f",
-        "passes": 435,
-        "fails": 0
-      }
-    }
-  }
-}
-```
-
-</CodeGroup>
+If you don't care about each individual metric measurement and instead want to see only the aggregated data, exporting the end-of-test summary data to a JSON file might be a better choice than using the JSON output described here. For more details, see `--export-summary` and `handleSummary()` in the [end-of-test summary docs](/results-visualization/end-of-test-summary).
```

Comment:
Can we then keep the `--export-summary` format from above in the new documentation about it ?